hudi-client/hudi-client-common/src/main/java/org/apache/hudi/callback/common/HoodieWriteCommitCallbackMessage.java

```diff
@@ -41,13 +41,19 @@
    */
   private String basePath;
 
+  /**
+   * partitionPath in hudi table
+   */
+  private String partitionPath;
+
   public HoodieWriteCommitCallbackMessage() {
   }
 
-  public HoodieWriteCommitCallbackMessage(String commitTime, String tableName, String basePath) {
+  public HoodieWriteCommitCallbackMessage(String commitTime, String tableName, String basePath, String partitionPath) {
```

Comment:
since we may need more information to send downstream in the future. maybe we can add a configuration for the user to set the fileds they need. then we can avoid change the constructors time by time

Reply:
good, let me think about it

Reply:
IMO, we can define a DTO to encapsulate more and more metadata information. wdyt?